pkg/services/authz/client.go

```diff
@@ -103,15 +102,13 @@ func newInProcLegacyClient(server *legacyServer) (authzlib.MultiTenantClient, er
 	)
 
 	return authzlib.NewLegacyClient(
-		&authzlib.MultiTenantClientConfig{},
+		&authzlib.ClientConfig{},
 		authzlib.WithGrpcConnectionLCOption(channel),
-		// nolint:staticcheck
-		authzlib.WithNamespaceFormatterLCOption(claims.OrgNamespaceFormatter),
```

Comment:
I haven't found a corresponding function so I assume it's not used anymore.

Reply:
Yes, it's actually another interceptor.